Certificate issue for validation webhook when deploying operator with helm chart

Run `make helm-install` or `make helm-generate`


```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "vharbor.kb.io": Post https://webhook-service.harbor-operator-ns.svc:443/validate-goharbor-io-v1alpha2-harbor?timeout=30s: service "webhook-service" not found
```

Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "vharbor.kb.io": Post https://harbor-operator.harbor-operator-ns.svc:443/validate-goharbor-io-v1alpha2-harbor?timeout=30s: x509: certificate signed by unknown authority
Makefile:406: recipe for target 'sample-harbor' failed

--- a/.github/workflows/build-dev.yml
+++ b/.github/workflows/build-dev.yml
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
+    - release-*
 
 jobs:
   docker:

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
     - master
+    - release-*
 
 jobs:
   # Dockerfile tests

--- a/config/helm/webhook/kustomization.yaml
+++ b/config/helm/webhook/kustomization.yaml
@@ -7,7 +7,7 @@ generatorOptions:
 
 commonAnnotations:
   # Trick so kustomize does not split the annotation value
-  cert-manager.io/cluster-issuer: '{{.Release.Namespace}}/{{. | include "chart.fullname"}}'
+  cert-manager.io/inject-ca-from: '{{.Release.Namespace}}/{{. | include "chart.fullname"}}-serving-cert'
 
 patchesStrategicMerge:
 - validatingwebhook_endpoint_patch.yaml

